templates/admin/admin_edit_listing.html

```diff
@@ -4,6 +4,12 @@
     <meta charset="UTF-8" />
     <meta http-equiv="X-UA-Compatible" content="IE=edge" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <style>
```

Comment:
Although this works, it's standard to keep our styling conventions in the Vue component it is being used